src/cpp/server/channelz/channelz_service.cc

```diff
@@ -25,6 +25,135 @@
 
 namespace grpc {
 
+Status ChannelzService::GetTopChannels(
```

Comment:
It looks like you've accidentally duplicated some code here.

Reply:
Ahh oops, was going to fast. Fixed